docs/services/ml/_index.md

```diff
@@ -27,46 +30,58 @@ Common use cases include:
 - Object classification, which enables machines to separate people, animals, plants, or other objects into predefined categories based on their characteristics, and to perform different actions based on the classes of objects.
 - Speech recognition, natural language processing, and speech synthesis, which enable machines to verbally communicate with us.
 
-Viam provides two services that together enable visual machine learning capabilities: the [ML model](/services/ml/deploy/) service and the [Computer Vision](/services/vision/) service.
+The Machine Learning (ML) model service allows you to deploy [machine learning models](/registry/ml-models/) to your machine.
+The service works with models trained inside and outside the Viam app:
+
+- You can [train](/how-tos/train-deploy-ml/) models on data from your machines.
+- You can upload externally trained models on the [**MODELS** tab](https://app.viam.com/data/models) in the **DATA** section of the Viam app.
+- You can use [ML models](https://app.viam.com/registry?type=ML+Model) from the [Viam Registry](https://app.viam.com/registry).
+- You can use a [model](/registry/ml-models/) trained outside the Viam platform whose files are on your machine.
+
+## Configuration
+
+You must deploy an ML model service to use machine learning models on your machines.
+Once you have deployed the ML model service, you can select an [ML model](#machine-learning-models-from-registry).
+
+After deploying your model, you need to configure an additional service to use the deployed model.
+For example, you can configure an [`mlmodel` vision service](/services/vision/) to visualize the predictions your model makes.
 For other use cases, consider [creating custom functionality with a module](/how-tos/create-module/).
 
-## Machine learning models
+{{<resources_svc api="rdk:service:mlmodel" type="ML model">}}
 
-Machine Learning (ML) models are mathematical models that can recognize patterns.
-Currently Viam supports TensorFlow Lite, TensorFlow, ONNX, and PyTorch.
+{{< alert title="Add support for other models" color="tip" >}}
+ML models must be designed in particular shapes to work with the `mlmodel` [classification](/services/vision/mlmodel/) or [detection](/services/vision/mlmodel/) model of Viam's [vision service](/services/vision/).
```

Comment:
Shapes? Is that the technical term?

Reply:
I think I copied this from somewhere, so not sure